src/main/java/org/cryptomator/common/settings/AbstractInvalidSetting.java

```diff
@@ -0,0 +1,23 @@
+package org.cryptomator.common.settings;
+
+import java.net.URI;
+
+public interface AbstractInvalidSetting {
```

Comment:
`Abstract` prefix suggests an abstract class. Avoid this on interfaces.

Reply:
Fixed by 714a0c6664cd4091fd607b85fb2153d4a82cebee